Gemfile

```diff
@@ -24,7 +24,7 @@ source 'https://gems.contribsys.com/' do
 end
 
 group :development, :test do
-  gem 'byebug', platforms: %i[mri mingw x64_mingw]
+  gem 'byebug', platforms: %i[mri windows]
```

Comment:
I bet you could dump the entire `platforms` bit here since we don't have any windows environments.